pkg/commands/testdata/workload-subPath.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 VMware, Inc.
```

Comment:
Issue: should we update the template https://github.com/vmware-tanzu/apps-cli-plugin/blob/baf4a47308c2b1615935a97010318d41864ac2a3/hack/boilerplate.go.txt
```
# Copyright 2021 VMware, Inc.
```